data/templates/ssh/sshd_config

```diff
@@ -15,10 +15,17 @@ HostKey {{ key }}{% endfor %}
 # https://infosec.mozilla.org/guidelines/openssh
 # ##############################################
 
-# Keys, ciphers and MACS
-KexAlgorithms curve25519-sha256@libssh.org,ecdh-sha2-nistp521,ecdh-sha2-nistp384,ecdh-sha2-nistp256,diffie-hellman-group-exchange-sha256
-Ciphers chacha20-poly1305@openssh.com,aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes192-ctr,aes128-ctr
-MACs hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512,hmac-sha2-256,umac-128@openssh.com
+{% if ssh_ciphers_compatibility == "intermediate" %}
+  KexAlgorithms diffie-hellman-group-exchange-sha256
```

Comment:
Re-reading this I'm wondering if this is really something people are going to use ... Mozilla's comment about it is : 

>This is mainly for use by RHEL6, CentOS6, etc. which run older versions of OpenSSH.

And I don't think that people in our userbase fall in that case...

However, it could be interesting to have something like `modern++` or `hardened` (or whatever name, idk) which increases stuff over the modern configuration, such as : 
- forbidding password logins (so only use asymetric keys)
- disabling root login on local network (assuming the admin is well aware of consequences about that)
- possibly removing ciphers with political speculation / superstition..., c.f. https://github.com/YunoHost/yunohost/pull/668